src/devices/UFireIse/UFireIse.cs

```diff
@@ -310,6 +310,12 @@ private float ReadFloat(Register register)
         private void WriteRegister(Register register, float data = 0)
         {
             // μFire ISE (Ion Specific Electrode) is 0 as default
+#if NETSTANDARD2_0
+            byte[] bytes = BitConverter.GetBytes(data);
+            ChangeRegister(register);
+            _device.Write(bytes);
+            DelayHelper.DelayMilliseconds(IseCommunicationDelay, allowThreadYield: true);
```

Comment:
This is no longer throwing the exceptions right?

Reply:
True, but that exception cannot really happen. The only situation where `TryWriteBytes` returns false is if there's not enough room in the buffer.